administrator/components/com_content/views/article/tmpl/edit.php

```diff
@@ -164,9 +164,9 @@
 			<?php echo JHtml::_('bootstrap.addTab', 'myTab', 'menulink', JText::_('COM_CONTENT_MENU_LINK', true)); ?>
 			<div class="row-fluid form-horizontal-desktop">
 				<div class="span6">
-					<?php echo $this->form->getControlGroup('menulink'); ?>
+					<?php echo $this->form->renderField('menulink'); ?>
 					<?php foreach ($this->form->getGroup('menulink') as $field) : ?>
-						<?php echo $field->getControlGroup(); ?>
+						<?php echo $field->renderField(); ?>
```

Comment:
No you should not ignore. Glad that you are not using deprecated method. Good work.